audio-worklet/src/bitcrusher-demo.js

```diff
@@ -31,35 +31,22 @@ class BitcrusherDemo {
     this.paramReduction_ =
         this.bitcrusherAudioWorklet_.parameters.get('reduction');
 
-    // GUI default settings depend on parameter values defined in audio-worklet.
-    this.bitDepthMaxValue_ = this.paramBitDepth_.maxValue;
-    this.bitDepthMinValue_ = this.paramBitDepth_.minValue;
-    this.bitDepthDefaultValue_ = this.paramBitDepth_.defaultValue;
-    this.reductionMaxValue_ = this.paramReduction_.maxValue;
-    this.reductionMinValue_ = this.paramReduction_.minValue;
-    this.reductionDefaultValue_ = this.paramReduction_.defaultValue;
-
-    // For the audio worklet, the bit depth and reduction parameters are
-    // modulated by changing the offset to connected constant source nodes.
-    this.bitDepthOffsetNode_ = new ConstantSourceNode(context, {offset:0});
-    this.reductionOffsetNode_ = new ConstantSourceNode(context, {offset:0});
-    this.bitDepthOffsetNode_.connect(this.paramBitDepth_);
-    this.reductionOffsetNode_.connect(this.paramReduction_);
-    this.bitDepthOffsetNode_.start();
-    this.reductionOffsetNode_.start();
-
-    // The user can swap between the bitcrusher and audio worklet node
+    // The script processor will have a delay relative to the audio worklet node
+    // proportional to the size of its buffersize. This delay can be heard as
+    // a glitch as the user switches node.
```

Comment:
The comment about glitching isn't really relevant to the description of this variable.  But oh well.